python/semantic_kernel/core_skills/planning/basic_planner.py

```diff
@@ -0,0 +1,203 @@
+from semantic_kernel.skill_definition import sk_function
+
+PROMPT = """
+You are a planner for the Semantic Kernel.
+Your job is to create a properly formatted JSON plan step by step, to satisfy the goal given.
+Create a list of subtasks based off the [GOAL] provided.
+Each subtask must be from within the [AVAILABLE FUNCTIONS] list. Do not use any functions that are not in the list.
+Base your decisions on which functions to use from the description and the name of the function.
+Sometimes, a function may take arguments. Provide them if necessary.
+The plan should be as short as possible.
+For example:
+
+[AVAILABLE FUNCTIONS]
+EmailConnector.LookupContactEmail
+description: looks up the a contact and retrieves their email address
+args:
+- name: the name to look up
+
+EmailConnector.EmailTo
+description: email the input text to a recipient
+args:
+- input: the text to email
+- recipient: the recipient's email address. Multiple addresses may be included if separated by ';'.
+
+LanguageHelpers.TranslateTo
+description: translate the input to another language
+args:
+- input: the text to translate
+- language: the language to translate to
+
+WriterSkill.Summarize
+description: summarize input text
+args:
+- input: the text to summarize
+
+FunSkill.Joke
+description: Generate a funny joke
+args:
+- input: the input to generate a joke about
+
+[GOAL]
+"Tell a joke about cars. Translate it to French"
+
+[OUTPUT]
+    {
+        "input": "cars",
+        "subtasks": [
+            {"function": "Joke"},
+            {"function": "TranslateTo", "args": {"language": "French"}}
+        ]
+    }
+
+[AVAILABLE FUNCTIONS]
+WriterSkill.Brainstorm
+description: Brainstorm ideas
+args:
+- input: the input to brainstorm about
+
+ShakespeareSkill.shakespeare
+description: Write in Shakespearean style
+args:
+- input: the input to write about
+
+WriterSkill.EmailTo
+description: Write an email to a recipient
+args:
+- input: the input to write about
+- recipient: the recipient's email address.
+
+[GOAL]
+"Tomorrow is Valentine's day. I need to come up with a few date ideas. She likes Shakespeare so write using his style.
+E-mail these ideas to my significant other"
+
+[OUTPUT]
+    {
+        "input": "Valentine's Day Date Ideas",
+        "subtasks": [
+            {"function": "WriterSkill.Brainstorm"},
+            {"function": "ShakespeareSkill.shakespeare"},
+            {"function": "WriterSkill.EmailTo", "args": {"recipient": "significant_other"}},
+            {"function": "translate", "args": {"language": "French"}}
+        ]
+    }
+
+[AVAILABLE FUNCTIONS]
+{{$available_functions}}
+
+[GOAL]
+{{$goal}}
+
+[OUTPUT]
+"""
+
+
+class BasicPlanner:
+    """
+    Description: Basic planner for the Semantic Kernel.
+
+    Usage:
+        kernel.import_skill(BasicPlanner(), "planner");
+
+    Examples:
+
+    {{planner.create_plan}}
+    {{planner.execute_plan}}
+    """
+
+    @sk_function(
```

Comment:
See https://github.com/microsoft/semantic-kernel/pull/523 -- we're moving away from Planner's being skills and instead are a top level concept in SK.

Reply:
Good to know! I wasn't able to register this planner as a skill anyway.

Reply:
@dluc if it's not going to be a skill, should I put this somewhere else? Maybe `python/semantic_kernel/planning`?

Reply:
For what it's worth, the dotnet stuff will be in nugetpackages like `Microsoft.SemanticKernel.Planning.SequentialPlanner` and the code is in `dotnet/src/Extensions/Planning.SequentialPlanner/` for example.